lib/common/schemaUtilsCommon.js

```diff
@@ -101,5 +129,11 @@ module.exports = {
       }
     }
     return min;
+  },
+
+  formatSchemaPathToConverToDataPath: function (schemaPath) {
```

Comment:
Can we improve naming here? Also, add description that explains what it's supposed to do? There is type as well seems like, did you mean `Convert` instead of `Conver`?

Reply:
Changed name of the method